integration_tests/cork_test.go

```diff
@@ -633,6 +624,148 @@ func (s *IntegrationTestSuite) TestCellarV2() {
 	})
 }
 
+func (s *IntegrationTestSuite) TestCellarV2_2() {
+	s.Run("Submit callOnAdaptor to MockCellar", func() {
+		s.checkCellarExists(vaultCellar)
+		s.waitForVotePeriod()
+
+		val := s.chain.validators[0]
+		kb, err := val.keyring()
+		s.Require().NoError(err)
+		clientCtx, err := s.chain.clientContext("tcp://localhost:26657", &kb, "val", val.keyInfo.GetAddress())
+		s.Require().NoError(err)
+
+		// Create the cork requests to send to Steward
+		cellarId := v2_2Cellar.Hex()
+
+		// Contains two adaptor calls, the first of which has two function calls inside of it, for a total of three function calls.
+		request := &steward_proto.SubmitRequest{
+			CellarId: cellarId,
+			CallData: &steward_proto.SubmitRequest_CellarV2Dot2{
+				CellarV2Dot2: &steward_proto.CellarV2_2{
+					CallType: &steward_proto.CellarV2_2_Multicall_{
+						Multicall: &steward_proto.CellarV2_2_Multicall{
+							FunctionCalls: []*steward_proto.CellarV2_2_FunctionCall{
+								{
+									Function: &steward_proto.CellarV2_2_FunctionCall_CallOnAdaptor{
+										CallOnAdaptor: &steward_proto.CellarV2_2_CallOnAdaptor{
+											Data: []*steward_proto.AdaptorCall{
+												{
+													Adaptor: adaptorContract.Hex(),
+													CallData: &steward_proto.AdaptorCall_SwapWithUniswapV1Calls{
+														SwapWithUniswapV1Calls: &steward_proto.SwapWithUniswapAdaptorV1Calls{
+															Calls: []*steward_proto.SwapWithUniswapAdaptorV1{
+																{
+																	Function: &steward_proto.SwapWithUniswapAdaptorV1_SwapWithUniV3_{
+																		SwapWithUniV3: &steward_proto.SwapWithUniswapAdaptorV1_SwapWithUniV3{
+																			Path:         []string{"0x1111111111111111111111111111111111111111", "0xAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"},
+																			PoolFees:     []uint32{1000, 2000},
+																			Amount:       "3",
```

Comment:
It seems like we're using the `Amount` and `AmountOutMin` to differentiate calls, and the two here have 3 and 4, but the one in the single function test above also uses 3 with the other values being the same. Perhaps we should make them all different to avoid reading the same one in the event feed when checking?